cdn/src/Server.ts

```diff
@@ -26,6 +27,8 @@ export class CDNServer extends Server {
 			res.set("Access-Control-Allow-Methods", req.header("Access-Control-Request-Methods") || "*");
 			next();
 		});
+		this.app.use(bodyParser.json({ inflate: true, limit: "10mb" }));
+		this.app.use(bodyParser.urlencoded({ inflate: true, limit: "10mb" }));
```

Comment:
If this gets removed, as it isn't needed, I'll merge it.

Reply:
If we need to remove, we need to close this PR.

Reply:
No we don't 🤨 @AlTech98 can update the PR